governance.md

```diff
@@ -84,7 +84,49 @@ The initial composition of the SC is self-defined and is made of 10 members:
 
 #### Selection
 
-The initial SC members will be responsible for defining how and when the SC membership can be expanded or renewed.
+The convention of the SC is for each organization that is significantly engaged
+in the project to be represented by one or two members.
+In addition to a company, an organization could also refer to a
+non-corporate institution such as school.
+Selection of the representatives of each organization is not within scope
+of the SC.
+The organization itself should select the individual(s) to represent it.
+Membership changes can fall into one of four categories,
+as described in the following sections.
+Each of these processes should be initiated by opening a PR against
+this document, explaining the motivations of the change and introducing
+any potential new members.
+Membership changes can be approved via GitHub and do not require an SC meeting.
+
+##### Expansion
+
+The SC can be expanded if a new organization begins making significant contributions
+to the project.
```

Comment:
At the moment, big users of the project are also contributors. I wonder if at some point we will welcome non-contributing users (the definition of contributors being "submitting PRs"). If a company starts using CC a lot, and provides valuable feedback in the form of issues, would that be enough for us to consider this a "contribution"?

Reply:
Speaking out of my mind here, based on my interpretation of the text, so bear with me. :-)

Your point comes from "(the definition of contributors being "submitting PRs").", but I do believe this is clarified a few lines down below, by the following sentence:
"There is no standard definition for significant contributions."

So, I do believe this will get into the "case by case analisys", but there's nothing in the text that would prevent those from being considered.

Reply:
Yes, contributions does not necessarily imply submitting PRs.